controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/horizon/TsdbQueryRewriter.java

```diff
@@ -66,7 +68,7 @@ private static void rewriteFilter(ObjectNode parent, Set<TenantName> tenantNames
         ObjectNode prev = ((ObjectNode) parent.get("filter"));
         ArrayNode filters;
         // If we dont already have a filter object, or the object that we have is not an AND filter
-        if (prev == null || !"Chain".equals(prev.get("type").asText()) || !"AND".equals(prev.get("op").asText())) {
+        if (prev == null || !"Chain".equals(prev.get("type").asText()) || prev.get("op") == null || !"AND".equals(prev.get("op").asText())) {
```

Comment:
I think `AND` is default, the documentation is... inconsistent: http://opentsdb.net/docs/3x/build/html/user_guide/semanticquery/filters.html#chain
> Combines one or more filters with a logical AND (the default) or a logical OR. 

But the table says default is `OR`

Logically, we want `AND` when doing suggestions, so I guess `AND` is default then since they are not specifying. In that case, we should change this to `!= null` and let the additional filters be appended to this chain.